src/sagemaker/image_uris.py

```diff
@@ -335,6 +336,11 @@ def _get_image_tag(
             if key in container_versions:
                 tag = "-".join([tag, container_versions[key]])
 
+    # Triton images don't have a trailing -gpu tag. Only -cpu images do.
```

Comment:
Because it's default option is gpu?

Reply:
@mohanasudhan Hi Mohan, that is correct - the default option is gpu.